src/backend/_scss/application.scss

```diff
@@ -522,39 +522,63 @@ a {
   min-height: 255px;
   position: absolute;
   transition: all 0.5s ease;
+  background-color:white;
```

Comment:
white is not allowed instead use $white. Here background-color seems over-written from 518